source/documentation/06-api-reference.md

```diff
@@ -187,8 +188,10 @@ These are the possible values of the card_brand parameter.
 
 ## API rate limits
 
-There is a maximum rate limit for requests to the API from your service account. The limit is high and most services are unlikely ever to exceed it.
+We rate limit POST requests to the API at 15 requests per second per government service. 
 
-If you do exceed the limit (that is, send a large number of requests in a short amount of time), you will receive P0900 errors. If this happens, you can attempt any rate-limited requests again after a second has passed.
+We also rate limit GET requests but currently at a very high level. We will publish an official limit for GET API requests in the next few weeks.
```

Comment:
In line with the other (minor) language recommendation:

> GET requests are also rate-limited, but at a very high level. In the future, we will publish an official rate limit for GET requests.

I'm a little reluctant to add the "in a few weeks" note because we can't necessarily commit to this exact timeframe and the documentation is full of holdover notes like that.

Reply:
Agree, it is safe to say in Future. Updated as per recommendation